src/CoreWlan/Enums.cs

```diff
@@ -116,4 +116,20 @@ public enum CWKeychainDomain : nuint_compat_int {
 		User = 1,
 		System = 2,
 	}
+
+	[Native]
+	public enum CWEventType : nuint_compat_int {
+		None = 0,
+		PowerDidChange = 1,
+		SSIDDidChange = 2,
+		BSSIDDidChange = 3,
+		CountryCodeDidChange = 4,
+		LinkDidChange = 5,
+		LinkQualityDidChange = 6,
+		ModeDidChange = 7,
+		ScanCacheUpdated = 8,
+		VirtualInterfaceStateChanged = 9,
+		RangingReportEvent = 10,
+		Unknown = int.MaxValue
```

Comment:
First this should be long.MaxValue in Unified, which can be accomplished doing something like this: https://github.com/xamarin/xamarin-macios/blob/master/src/UIKit/UIEnums.cs#L771-L775

Second I think our generated code will not work correctly when passing such an enum value as a parameter on 32-bit, because we'll try to cast the value to a 32-bit value, which will overflow and throw an exception. Using unchecked code isn't the solution either, since these are signed values, and `((int) long.MaxValue != int.MaxValue`, we'll need something more complex. But that's not a problem this PR should try to fix, so just making sure the enum has the right value should be good enough.